components/Form/index.js

```diff
@@ -18,26 +20,83 @@ const StyledSelect = styled.select`
   border-radius: 15px;
 `;
 
+const UploadPreview = styled(Image)`
+  width: 100px;
+`;
+
+const UploadSection = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 30px;
+`;
+
 export default function Form({ onSubmit, onCancel, project = {} }) {
+  const [imageSelected, setImageSelected] = useState("");
+  const [imageId, setImageId] = useState(project.image || "/sample-image.png");
+
+  const uploadImage = async () => {
+    const formData = new FormData();
+    formData.append("file", imageSelected);
+    formData.append("upload_preset", "diy-app");
+    try {
+      const res = await fetch(
+        `https://api.cloudinary.com/v1_1/dzxsogtiq/image/upload`,
+        {
+          method: "POST",
+          body: formData,
+        }
+      );
```

Comment:
You are using a secret for uploading images here in the frontend (the cloudinary endpoint). This is highly insecure. To avoid this issue, you should create an API Route for uploading images and hide these secrets behind this endpoint. Your frontend then uploads the image to your own API endpoint, which in turn handles the upload to cloudinary. By this, the secrets stay on your server and the enduser never sees them.